<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtlantTabWidget</class>
 <widget class="QWidget" name="AtlantTabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="modeCB">
       <item>
        <property name="text">
         <string>Adaptation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Watch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Check</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Work</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="freqSB">
       <property name="minimum">
        <double>800.000000000000000</double>
       </property>
       <property name="maximum">
        <double>3000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="widthSB">
       <property name="maximum">
        <double>20.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendPB">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="logsTE">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
